realm/realm-library/src/main/java/io/realm/internal/TableView.java

```diff
@@ -510,8 +510,9 @@ public long upperBoundLong(long columnIndex, long value) {
     public TableView findAllLong(long columnIndex, long value){
         // Execute the disposal of abandoned realm objects each time a new realm object is created
         context.executeDelayedDisposal();
-        long nativeViewPtr = nativeFindAllInt(nativePtr, columnIndex, value);
-        try { 
+        long nativeViewPtr = 0;
+        try {
+            nativeViewPtr = nativeFindAllInt(nativePtr, columnIndex, value);
```

Comment:
Hmm, not sure I understand that because `long` can never be `null`, did you mean `0`?